fix(curriculum): Recipe Tracker improvements (#57694)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/25-front-end-development/workshop-recipe-tracker/66fbcf750a62784cf11f5631.md

Now, you should work on calculating the `averageRating`, `totalIngredients`, and the `difficultyLevel` for each recipe in the `recipes` array.

> Start by creating a `getAverageRating` function that has single parameter, accepting an array with ratings. Inside the function, calculate the average of ratings from array passed to function.

Reviewer: ```suggestion
Start by creating a `getAverageRating` function that has single parameter, accepting an array with ratings. Inside the function, calculate the average rating from the array passed to the function.
```

> Start by creating a `getAverageRating` function that has single parameter, accepting an array with ratings. Inside the function, calculate the average rating from the array passed to the function.

Reviewer 2: ```suggestion
Start by creating a `getAverageRating` function that takes a single argument, which is an array with ratings. Inside the function, calculate the average rating from the array passed to the function.
```
